convert2rhel/redhatrelease.py

```diff
@@ -89,7 +95,19 @@ def _write_altered_yum_conf(self):
     def get_yum_conf_filepath():
         return YumConf._yum_conf_path
 
+    @staticmethod
+    def is_modified():
+        """Return true if the YUM/DNF configuration file has been modified by the user."""
+        if pkgmanager.TYPE == "yum":
+            conf = "/etc/yum.conf"
+        else:
+            conf = "/etc/dnf/dnf.conf"
+
+        output, _ = utils.run_subprocess("rpm -Vf %s" % conf, print_output=False)
+        # rpm -Vf does not return information about the queried file but about all files owned by the rpm
+        # that owns the queried file. Character '5' on position 3 means that the file was modified.
+        return True if re.search(r".{2}5.*? %s$" % conf, output, re.MULTILINE) else False
```

Comment:
Is there any reason for re.MULTILINE flag?
then you use ".", but in regex this means any character. If you want `.` character specifically, then you should use `\.`
Also, you're only interested in this 5 at 3rd position. So maybe it will be simpler to solve it as:
return True if output[2] == "5" else False?

But if you want regex, you also can simplify it:
return True if re.search(r"\.{2}(5).+", "..5....T.  c /etc/dnf/dnf.conf") else False

Reply:
I think, that re.MULTILINE is necessary because the output from `rpm -Vf` can have more lines. The dots there are ok, there can be any characters. 

This part gave me @bocekm, so maybe he'll correct me if I'm saying something wrong. It was after a research because I had some idea, but it showed up as a little bit more complicated as we expected :)

Reply:
I think -Vf gives the output only for the given file. but I might be wrong. I tried to modify those files, but always get only one line. @bocekm should know more about rpm utility.

Reply:
Against all common sense, `rpm -Vf <file>` outputs all changed files of the package to which `<file>` belongs. So the multiline is correct.

Reply:
```
# touch /etc/yum.conf
# touch /etc/yum/version-groups.conf
# rpm -Va yum
.......T.  c /etc/yum.conf
.......T.  c /etc/yum/version-groups.conf
# rpm -Vf /etc/yum.conf
.......T.  c /etc/yum.conf
.......T.  c /etc/yum/version-groups.conf
```

Reply:
ok, then do not forget a `^` in the regex, but the filename should be there

Reply:
```suggestion
        return True if re.search(r"^.{2}5.*? %s$" % conf, output, re.MULTILINE) else False
```